### PR TITLE
Feature/Enable tooltips on Mouse hover in flutter web/desktop

### LINKED ISSUE #275: Show tooltip on MouseOver on Web/Desktop
This was discussed a bit here: https://github.com/imaNNeoFighT/fl_chart/issues/37#issuecomment-517467347 but it was never implemented although the poster claimed it was working.

### PR DESCRIPTION
**Pull request by shripal17**
Partially closes #275 

1. Implemented [MouseRegion](https://api.flutter.dev/flutter/widgets/MouseRegion-class.html) only for line chart, bar chart and pie chart (Scatter chart left untouched)
2. Tested only in flutter web: Chrome 87.0.4280.141 (Official Build) (x86_64) on Mac OS Big Sur 11.1
3. Also added flutter web execution to example app
4. Fix SVG in pie chart badge example not loading in web
5. Added Next/Previous FABs to example app for web and disabled scrolling in PageView for web

### CODE REVIEW

**Comment by imaNNeoFighT** [comment]
Hi,
Thanks for your great merge request.
I forgot to write in CONTRIBUTIONS.md that you must set your merge request target to the `dev` branch.
I changed the target to `dev` and now there are a lot of conflicts (we made some dartfmt changes on the `dev` branch)
BTW I apologize for that. I am reviewing your changes.
Please fix the conflicts.
Thanks!

**Comment by imaNNeoFighT** [comment]
I reviewed all your changes smoothly.
I have to say that you are awesome. 
Everything handled very well. I like that. Hope to see more from you :))

Thanks!

File: lib/src/chart/bar_chart/bar_chart_painter.dart [hunk 1]
```diff
@@ -75,7 +75,7 @@ class BarChartPainter extends AxisChartPainter<BarChartData> with TouchHandler<B
       Size viewSize, List<BarChartGroupData> barGroups, BarChartAlignment alignment) {
     final Size drawSize = getChartUsableDrawSize(viewSize);
 
-    final List<double> groupsX = List<double>(barGroups.length);
+    final List<double> groupsX = List.filled(barGroups.length, 0, growable: false);
```
Comment (imaNNeoFighT) [inline lib/src/chart/bar_chart/bar_chart_painter.dart:78]: Please revert this change, because I realized someone else changed this in a separate merge request #540.

**Comment by shripal17** [comment]
Done, thanks!

**Comment by shripal17** [comment]
by the way, i just noticed line_chart.dart and bar_chart.dart import cupertino lib, and not material lib, is that by mistake or intended?

**Comment by imaNNeoFighT** [comment]
It is by mistake.

File: lib/src/chart/bar_chart/bar_chart_painter.dart [hunk 2]
```diff
@@ -1,795 +1,795 @@
-import 'dart:math';
```
Comment (imaNNeoFighT) [inline lib/src/chart/bar_chart/bar_chart_painter.dart:1]: Please revert this file.

**Comment by shripal17** [comment]
@imaNNeoFighT done

**Comment by imaNNeoFighT** [comment]
Thanks!
